spra-api/shared/src/main/scala/net/wiringbits/spra/api/AdminDataExplorerApiClient.scala

```diff
@@ -142,7 +142,8 @@ object AdminDataExplorerApiClient {
 
       prepareRequest[AdminCreateTable.Response]
         .post(uri)
-        .body(Json.toJson(request).toString())
+        // TODO: check the AdminCreateTable.Request model, it isn't parsing correctly to json
```

Comment:
Is this still an issue?

Reply:
Yes, with `Json.toJson(request)` it throws `Json validation error List((obj.data,List(JsonValidationError(List(error.expected.jsstring),ArraySeq())))).`